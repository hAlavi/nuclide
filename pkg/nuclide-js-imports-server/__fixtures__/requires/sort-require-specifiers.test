

const {aB, aA, C, B} = require('Z');
const {s, Q, P} = require('X');

aA(aB(B, C, D))
s(Q, P)

// @nolint
