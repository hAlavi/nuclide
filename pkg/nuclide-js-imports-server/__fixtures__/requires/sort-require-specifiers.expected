const D = require('D');
const {P, Q, s} = require('X');
const {B, C, aA, aB} = require('Z');

aA(aB(B, C, D))
s(Q, P)

// @nolint
